OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -1027,7 +1027,8 @@ void TryToUseSupportPower(Actor self)
 					// Valid target found, delay by a few ticks to avoid rescanning before power fires via order
 					BotDebug("AI: {2} found new target location {0} for support power {1}.", attackLocation, sp.Info.OrderName, Player.PlayerName);
 					waitingPowers[sp] += 10;
-					QueueOrder(new Order(sp.Info.OrderName, supportPowerMngr.Self, false) { TargetLocation = attackLocation.Value, SuppressVisualFeedback = true });
+					var activateOrderName = sp.Info.AllowMultiple ? sp.Info.OrderName + "_" + sp.Instances.First().Self.ActorID : sp.Info.OrderName;
```

Comment:
If AllowMultiple is enabled then each support power will only have a single instance in the list.